packages/main/src/Popover.ts

```diff
@@ -21,6 +21,8 @@ import PopoverTemplate from "./PopoverTemplate.js";
 import PopupsCommonCss from "./generated/themes/PopupsCommon.css.js";
 import PopoverCss from "./generated/themes/Popover.css.js";
 
+type Opener = HTMLElement | string | null | undefined;
```

Comment:
This type describes public API and it will be good to be exported or to be used as a union type like on the other places.  In documentation as well it appears as **Opener** which is not much clear what could be used on first sight :

<img width="311" height="331" alt="Image" src="https://github.com/user-attachments/assets/076df0d7-b917-4862-9794-faf9e8c47cb2" />